src/DIRAC/Resources/Computing/HTCondorCEComputingElement.py

```diff
@@ -237,6 +251,40 @@ def _reset(self):
         self.log.debug("Remote scheduler option:", self.remoteScheddOptions)
         return S_OK()
 
+    def _executeCondorCommand(self, cmd, keepTokenFile=False):
+
+        if self.token:
+            # Create a new token file if we do not keep it across several calls
+            tFile = None
+            if keepTokenFile:
+                tFile = self.tokenFile
+            if not tFile:
+                fd, tFile = tempfile.mkstemp(suffix=".token", prefix="HTCondorCE_", dir=self.workingDirectory)
+                writeToTokenFile(self.token["access_token"], tFile)
+
+            htcEnv = {
+                "_CONDOR_SEC_CLIENT_AUTHENTICATION_METHODS": "SCITOKENS",
+                "_CONDOR_SCITOKENS_FILE": tFile,
+            }
+        else:
+            htcEnv = {"_CONDOR_SEC_CLIENT_AUTHENTICATION_METHODS": "GSI"}
+
+        result = executeGridCommand(
+            self.proxy,
+            cmd,
+            gridEnvScript=self.gridEnv,
+            gridEnvDict=htcEnv,
+        )
+        # Remove token file if we do not want to keep it
+        if tFile:
```

Comment:
If `self.token` is not defined here `tFile` is not initialized and does not exist.
`tFile=None` should be moved before `self.token`.